scripts/build.prod.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-version="$(cat ../client/package.json | grep -i "version*" | awk -F '"' '{print $4}')" 
+version="$(cat ./client/package.json | grep -i "version*" | awk -F '"' '{print $4}')" 
```

Comment:
Line 24 (`dirname ${BASH_SOURCE[0]}`) in this file detects current path and makes the script robust to the changes in cwd.
Can you use the same trick here, so we don't make any assumption about how build is started?

Reply:
I've solved it with another trick with git.